### ISSUE TITLE
Text is drawn when looking in opposite direction with camera

### ISSUE THREAD

**Issue by bvssvni** [PistonDevelopers/gfx_debug_draw#50]
It should only draw text when looking at the coordinates. Currently it draws in both direction. When running the hello_world example in [turbine](https://github.com/pistondevelopers/turbine) the "X", "Y", "Z" letters are displayed when looking away from origo.

**Comment by phrohdoh** [comment]
I figured this was intentional so did not file anything about it.
Is it safe to assume that things of this nature are probably to be polished?

**Comment by bvssvni** [comment]
Maybe this should be moved to https://github.com/pistondevelopers/gfx_text. I thought it was possible to do the filtering here since there was a loop, but it turns out it just adds the stuff from the singleton.

**Comment by bvssvni** [comment]
@Phrohdoh I do not think it is intentional. The text gets in the way for other information.
